changes/23070-install-execution-status

```diff
@@ -0,0 +1 @@
+* Pass through original install status (rather than blank) when retrieving install details from the activity feed when the installer package has been updated or the software has since been removed from the host
```

Comment:
Please use past tense.

Reply:
Done in [fcd04b4](https://github.com/fleetdm/fleet/pull/23072/commits/fcd04b48aa76d42aee59ab897a67c0566a8fc059)